src/App.tsx

```diff
@@ -97,6 +97,7 @@ function App() {
         <table>
           <thead>
             <tr>
+              <th>Position</th>
```

Comment:
What do you think about adding the position into the "Player" column instead of adding a new one? Or would that look bad since all the names are different lengths? I could go either way on this

One thing I will note is that the extra column does kinda break mobile